test/test_jit.py

```diff
@@ -7355,6 +7357,54 @@ def forward(self, x, y, c):
         ge_graph = jit_trace.__getattr__('forward').graph_for(x, y, c)
         self.assertExpectedGraph(ge_graph, 'jit')
 
+    def test_exceptions(self):
+        cu = torch.jit.CompilationUnit('''
+            def foo(cond):
+                if bool(cond):
+                    raise ValueError(3)
+                return 1
+        ''')
+
+        cu.foo(torch.tensor(0))
+        with self.assertRaisesRegex(RuntimeError, "Exception"):
```

Comment:
Can we at least throw a `JITException`? That will be a clearer change than defaulting to `RuntimeError` for anything that was written inside.